terraso_backend/apps/project_management/migrations/0003_site_created_by.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 4.2.1 on 2023-05-23 05:43
+
+import django.db.models.deletion
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ("project_management", "0001_squashed_0002_project_and_more"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="site",
+            name="created_by",
+            field=models.ForeignKey(
+                blank=True,
+                null=True,
+                on_delete=django.db.models.deletion.RESTRICT,
+                to=settings.AUTH_USER_MODEL,
+                verbose_name="user who created the site",
+            ),
+        ),
```

Comment:
question: i don't see a `created_by` field on the `site` model, is this migration up to date?

Reply:
Fixed by removing the migration